src/Custom/Responses/Streaming/StreamingResponseImageGenerationCallCompletedUpdate.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OpenAI.Responses;
+
+// CUSTOM: Renamed.
+[CodeGenType("ResponseImageGenCallCompletedEvent")]
+public partial class StreamingResponseImageGenerationCallCompletedUpdate
```

Comment:
nit: Can we group these and other simple rename attributes like this into a single GeneratorStubs.cs file under the Responses folder?